include/nbl/builtin/hlsl/lod_library/structs.hlsl

```diff
@@ -0,0 +1,64 @@
+
+#ifndef _NBL_GLSL_LOD_LIBRARY_STRUCTS_GLSL_INCLUDED_
+#define _NBL_GLSL_LOD_LIBRARY_STRUCTS_GLSL_INCLUDED_
+
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace lod_library
+{
```

Comment:
delete the file, we'll redo the entire example anyway
(also delete the GLSL while you're at it)